cmd/minishift/cmd/root.go

```diff
@@ -213,14 +222,37 @@ func initializeProfile() string {
 		activeProfile string
 	)
 
+	// If `minishift profile set` or `minishift start --profile` command is used.
```

Comment:
We are not doing anything related to `start` command in this function, so the comment is confusing to me.